graphql-schemas/schema.yml

```diff
@@ -1430,6 +1431,16 @@ confs:
   - { name: region, type: string, isRequired: true }
   - { name: subnets, type: AWSSubnet_v1, isList: true }
 
+- name: AWSManagedVPC_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: labels, type: json }
+  - { name: name, type: string, isRequired: true }
+  - { name: description, type: string }
+  - { name: network, type: Network_v1, isRequired: true }
+  - { name: account, type: AWSAccount_v1, isRequired: true }
```

Comment:
We need vpc_id to be optional for integrations that depending on it.

Reply:
subnets field too. terraform_tgw_attachments depends on it

Reply:
Sorry, haven't we decide to create a new schema that doesn't need vpc_id and fetch them from AWS by using a data source in the integrations that needs them?

Reply:
You are right. vpc_id isn't needed. That discussion reminded it for me.